### ISSUE TITLE
Is it possible to integrate this with a hapijs server?

### ISSUE THREAD

**Issue by Iodine-** [bluejamesbond/Scribe.js#79]
I see all the express hooks - just wondering if/how much would it take to have this running on a hapijs application

**Comment by bluejamesbond** [comment]
I don't think it is compatible with it. I haven't used HapiJS. However, it is fairly reasonable to make a transform for it to support with HapiJS to make in the alpha version
